### PR TITLE
Contribution guideline

### PR DESCRIPTION
**Pull request by hybridcattt**
Contribution guideline contains some information that is supposed to help get started with the project. 
Tried to write down everything that was discussed and decided so far. 
Let me know if I missed anything!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Say Their Name iOS Contribution Guide
+
+Welcome! Thank you for being here and for willing to contribute to the cause.
+
+Below you will find some information about the project that will help
```
Comment (Kilo-Loco) [inline CONTRIBUTING.md:5]: ... that will help you ...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Git workflow
+
+The main branch is `development`, where all ongoing development work converges.
+`master` branch is reserved for some more finished version of the app.
+
+When working on something, create a new branch from `development`, and make a PR
+back into `development` branch.
+
+There are multiple people working from different timezones, so it's good to try
+to keep your branch up-to-date by rebasing it on top of latest `development`.
```
Comment (Kilo-Loco) [inline CONTRIBUTING.md:24]: YES!!! 😁

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+The best approach is to make smaller PRs: for example, laying some foundation for
+a feature, or refactoring, can be PR'd before continuing further with feature
+development.
+
+### Making a PR
+
+When making a PR, add a meaningful title and description.
+
+If any development is done on the UI (additions or non-trivial changes), please
+include a screenshot of the change.
```
Comment (Kilo-Loco) [inline CONTRIBUTING.md:36]: THANK YOU FOR INCLUDING THIS!!

File: CONTRIBUTING.md (hunk 1 above)
Comment (hybridcattt) [reply CONTRIBUTING.md:5]: will fix!

**Comment by ohwhen** [comment]
This is great, nice work @hybridcattt

**Pull request merged**